java/core/src/main/java/co/worklytics/psoxy/gateway/impl/CachingConfigServiceDecorator.java

```diff
@@ -24,9 +26,11 @@ public class CachingConfigServiceDecorator implements ConfigService {
 
     private final Object $writeLock = new Object[0];
 
-    private final String NEGATIVE_VALUE = "##NO_VALUE##";
+    @VisibleForTesting
+    static final String NEGATIVE_VALUE = "##NO_VALUE##";
```

Comment:
Just personal taste, but each time I see this I'm thinking in a *pure negative* value (<0); I understand that maybe what we want to represent is a value not defined in cache, so maybe... ?

```suggestion
    static final String NO_CACHE_VALUE = "##NO_VALUE##";
```